vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -0,0 +1,53 @@
+# Entando Glossary
+
+## Entando Terms
+
+| Term |  Description
+|:--|:--
+| Entando Application | An application built with the Entando Platform and available at a given URL |
+| Entando App Builder | The frontend of the Entando Platform that hosts the Entando WCMS and provides a feature-rich user interface to configure and interact with components, design and create pages, manage content, and build applications |
+| Entando App Engine | The core runtime engine responsible for the primary out-of-the-box services required to develop Entando Applications; it exposes the Entando App Builder APIs, assembles and coordinates components within the Entando App Builder, manages the cluster infrastructure, and provides the data access layer to persist the page and application design |
```

Comment:
We should remove "manages the cluster infrastructure". The Engine doesn't modify k8s.